### PR TITLE
Agent 0.3

### PR DESCRIPTION
**Pull request by harshjv**

### CODE REVIEW

File: heroku.config.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[database]
+uri = "mongodb+srv://satoshipro:6mVsstEk2fP2TFds@cluster0.rhqsi.mongodb.net/liquality_mainnet?retryWrites=true&w=majority"
```
Comment (monokh) [inline heroku.config.toml:2]: Did you mean to check this one in?

Comment (harshjv) [reply heroku.config.toml:2]: ah. my bad. terminated mongo instance. will remove the file.

**Review by monokh — COMMENTED** [review]
I see you have added the fee providers, but there is no call to `getFees()` to set the fee on txs
assuming this is separate

**Comment by harshjv** [comment]
@monokh it uses `getFees()` for transactions within Order model. https://github.com/liquality/atomicagent/blob/dev/src/models/Order.js#L432-L441

**Pull request merged**